all2vec/__init__.py

```diff
@@ -217,21 +217,28 @@ def save(self, folder):
             json.dump(enttypes, handle)
 
     @classmethod
-    def load(cls, folder):
-        """Load object."""
+    def load(cls, file_getter_or_folder):
+        """Load object.
+
+        file_getter_or_folder -- a FileGetter or a folder string
+        """
+        # to preserve backwards compatibility
+        if isinstance(file_getter_or_folder, str):
```

Comment:
Dumb Python 2 fact: this will fail for Unicode strings! 

```
In [3]: s = u'Hey Sam!'

In [4]: isinstance(s, str)
Out[4]: False
```

I'd suggest using `basestring` for this comparison.